ui/templates/base.html

```diff
@@ -10,7 +13,7 @@
   <body>
     {% block content %}
     {% endblock %}
-  <script type="text/javascript" src="{{ outer_src }}"></script>
+  <script type="text/javascript" src="{{ inner_src }}"></script>
```

Comment:
OK, I've moved `outer_src` to `index.html` and `inner_src` to `dashboard.html`. Make sense @noisecapella? Thanks.